app/src/modules/network/conn_info.cddl

```diff
@@ -1,13 +1,24 @@
 ; CDDL schema to encode the object defined in https://github.com/hello-nrfcloud/proto-map/blob/saga/lwm2m/14203.xml
 ; depends on lwm2m_senml definitions
 
+base-attributes = {
+	bn => "14203/0/",                   ; Connection Information object
+	bt => 1700000000...3000000000       ; UNIX timestamp in seconds
+}
+
+rsrp = {
+	n => "2",                           ; RSRP resource ID
+	vi => int,                          ; RSRP (in dBm)
```

Comment:
```suggestion
	vi => -140..-40,                          ; RSRP (in dBm)
```

See https://docs.nordicsemi.com/bundle/ref_at_commands/page/REF/at_commands/mob_termination_ctrl_status/proc_cesq_set.html

Reply:
I have used `-157..-44` because that seems to be range supported by the 91x1 devices as per https://docs.nordicsemi.com/bundle/ref_at_commands_nrf91x1/page/REF/at_commands/mob_termination_ctrl_status/coneval_set.html#ariaid-title1